server/src/test/java/com/cloud/user/AccountManagetImplTestBase.java

```diff
@@ -73,6 +73,7 @@
 import com.cloud.user.dao.SSHKeyPairDao;
 import com.cloud.user.dao.UserAccountDao;
 import com.cloud.user.dao.UserDao;
+import com.cloud.user.dao.UserDataDao;
```

Comment:
looks like these are extra changes :) please remove them.

Reply:
thanks @harikrishna-patnala for review

the unit test in AccountManagerTest did not pass without it :-(